core/textinput/CMakeLists.txt

```diff
@@ -30,9 +30,9 @@ ROOT_OBJECT_LIBRARY(TextInput
   src/textinput/TerminalDisplayWin.cpp
   src/textinput/TextInputContext.cpp
   src/textinput/TextInput.cpp
+  INCLUDES
+    src
```

Comment:
I am bit worried here, do we really need 'src' here?

Reply:
It was in old cmake file, one could test if it required.